petastorm/tests/test_end_to_end.py

```diff
@@ -89,24 +119,24 @@ def test_simple_read_with_disk_cache(synthetic_dataset, reader_factory, tmpdir):
     with reader_factory(synthetic_dataset.url, num_epochs=2,
                         cache_type='local-disk', cache_location=tmpdir.strpath,
                         cache_size_limit=CACHE_SIZE, cache_row_size_estimate=ROW_SIZE_BYTES) as reader:
-        _check_simple_reader(reader, synthetic_dataset.data, 200)
+        list(reader)
```

Comment:
this no longer checks the ids being read, I understand that this test is more to check that we can read with the disk cache without failures but is there a reason not to check that the ids make sense as well?

Reply:
I removed it because _check_simple_reader does not support batches. Replaced it with the test on ids. That's concise enough.

Reply:
Added an id check. Checking all fields is a bit trickier (though possible). Do you think it's needed?

Reply:
I dont think its necessary. I think that this test doesnt need to cover that logic.